frontend/src/components/regions/CardDetails/Review.js

```diff
@@ -68,26 +80,43 @@ const Review = ({ review }) => {
         className={classes.cardHeader}
       />
       <CardContent className={classes.cardContent}>
-        {/* <Typography className={classes.cardFont}>
-          <ShowMoreText
-            lines={1}
-            more={<ExpandMore />}
-            less={<ExpandLess />}
-            onClick={onClick}
-            expanded={expand}
-            width={0}
-            truncatedEndingComponent={"..."}
-            > */}
-        <Markdown children={review.comments} />
-        {/* </ShowMoreText> */}
-        {/* </Typography> */}
+        {/* 
+            217 is the approximate max number of chracters that can be visible before the need for
+            a scroll. As long as a review is longer than that, show "...read more"
+        */}
+        {review.comments.length > 220 ? (
+          <div>
+            <Button
+              type="button"
+              size="small"
+              className={classes.iconAlignment}
+              onClick={() => setOpenReviewPopUp(true)}
+            >
+              <Typography variant="caption" className={classes.readMoreStyle}>
+                ... Read more
+              </Typography>
+            </Button>
+            <Typography noWrap>
+              {cleanMarkdown(review.comments.trim())}
```

Comment:
Here you are just making the review less readable.  Long reviews will appear as a wall of text. That is not nice.

Before you were actually attempting to shorten it.

Please dont throw away that functionality. RAther fix it. 

```
function trimReviewComments({comments}){
    return cleanMarkDown(comments.trim()replace('\n',' ).slice(0,220))
}
```


Then in your component, we should be consistent in how things get displayed. 

```
 <CardContent className={classes.cardContent}>

<div>
            <Button
              type="button"
              size="small"
              className={classes.iconAlignment}
              onClick={() => setOpenReviewPopUp(true)}
            >
              <Typography variant="caption" className={classes.readMoreStyle}>
                ... Read more
              </Typography>
            </Button>
            <Typography noWrap>
                {trimReviewComments(comments)}
           </Typography>
    </div>
</CardContent>
```